nss-day-cohort-60/rare-client-glorious-gitastrophies
# Jl add post

### Add POST request to /posts

## To Test:
1. pull jl-add-post branch
2. checkout to jl-add-post branch
3. navigate to newPost using the navbar
4. fill out the form
5. you should auto-navigate to PostList with your new post visible in the list.
